spec/daru/io/exporters/sql_spec.rb

```diff
@@ -6,16 +6,11 @@
   let(:prepared_query) { double }
 
   it 'writes to an SQL table' do
-    expect(dbh)
-      .to receive(:prepare)
+    allow(dbh).to receive(:prepare)
```

Comment:
Rubocop-rspec prefers to check for `receive` with spy / `allow` rather than `expect`.